tasks/icinga2-RedHat.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: RedHat - Add Icinga 2 repository
+  become: yes
+  copy: src=ICINGA-release.repo dest=/etc/yum.repos.d/ICINGA-release.repo
+  when: i2_use_repository
```

Comment:
It is better to build the file from a template with variables (same as done for the Debian) 
and missing a new line

Reply:
You mean it would be better to add `i2_yum_key` and `i2_yum_url` ansible variables rather than relying on the `$releasever` yum variable? I think you're right, it would be unified with the apt stuff and could be overwritten by the user.